lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -203,6 +203,11 @@ class AwsCompileStreamEvents {
               streamResource.Properties.MaximumRetryAttempts = event.stream.maximumRetryAttempts;
             }
 
+            if (event.stream.bisectBatchOnFunctionError) {
+              streamResource.Properties.BisectBatchOnFunctionError =
+                event.stream.bisectBatchOnFunctionError;
```

Comment:
It'll be good to ensure in all cases it's a _boolean_ that ends in CF template, so I'd just assign `true`

Reply:
Thanks for the review! That sounds good, I'll make the adjustment 👍